### PR TITLE
Remove chainlink_deployer.js, skip truffle migrations in test

### PR DESCRIPTION
**Pull request by dimroc**
With such a small reduced chainlinkDeployer, do we even need it? We
can easily replace it with a direct call to `request.post(...)`.

### CODE REVIEW

File: examples/echo_server/migrations/5_run_log.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+let url = "http://chainlink:twochains@localhost:6688/v2/specs";
+let job = {
+  "_comment": "A runlog has a jobid baked into the contract so chainlink knows which job to run.",
+  "initiators": [{ "type": "runlog" }],
+  "tasks": [
+    { "type": "HttpPost", "url": "http://localhost:6690" }
+  ]
 };
+
+module.exports = clmigration(async function(truffleDeployer) {
```
Comment (dimroc) [inline examples/echo_server/migrations/5_run_log.js]: Please note that only migrations wrapped with `clmigration` are skipped in the test environment. I didn't wrap all migrations, just ones reaching out to external APIs and any migrations running afterwards. This ended up being 4 total across all examples and solidity.

File: examples/uptime_sla/migrations/4_uptime_sla.js [hunk 2]
```diff
@@ -1,22 +1,40 @@
...
-let Oracle = artifacts.require("../../../solidity/contracts/Oracle.sol");
-let LINK = artifacts.require("../../../solidity/contracts/LinkToken.sol");
+const clmigration = require("../clmigration.js");
+const request = require("request-promise");
+const UptimeSLA = artifacts.require("./UptimeSLA.sol");
+const Oracle = artifacts.require("../../../solidity/contracts/Oracle.sol");
+const LINK = artifacts.require("../../../solidity/contracts/LinkToken.sol");
 
-module.exports = function(truffleDeployer) {
+let url = "http://chainlink:twochains@localhost:6688/v2/specs";
```
Comment (j16r) [inline examples/uptime_sla/migrations/4_uptime_sla.js]: Should we be using: `"http://"+process.env.USERNAME+":"+process.env.PASSWORD+"@localhost:6688/v2/specs"`

**Comment by j16r** [comment]
`easily replace it with a direct call to request.post(...)` 👍

File: examples/uptime_sla/migrations/4_uptime_sla.js (hunk 2 above)
Comment (thodges-gh) [reply examples/uptime_sla/migrations/4_uptime_sla.js]: I was hoping to move away from hard-coding defaults and using whatever system's values are set for $USERNAME and $PASSWORD instead. However, I don't mind going back to chainlink:twochains.

**Pull request merged**